go/cmd/vttablet/vttablet.go

```diff
@@ -38,6 +41,7 @@ var (
 	enforceTableACLConfig = flag.Bool("enforce-tableacl-config", false, "if this flag is true, vttablet will fail to start if a valid tableacl config does not exist")
 	tableACLConfig        = flag.String("table-acl-config", "", "path to table access checker config file; send SIGHUP to reload this file")
 	tabletPath            = flag.String("tablet-path", "", "tablet alias")
+	tabletBackupOnly      = flag.Bool("tablet_backup_only", false, "if set this tablet will come on line and then take a backup then stop")
```

Comment:
Is it necessary to prefix the flag name with tablet? What about something like `backup_and_exit`?

Reply:
I was just building off `tablet-path`

Reply:
there are tons of vttablet options that are _not_ prefixed with tablet.